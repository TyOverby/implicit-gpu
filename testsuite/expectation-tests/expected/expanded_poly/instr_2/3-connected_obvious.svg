version https://git-lfs.github.com/spec/v1
oid sha256:f475f35cedfde6f085145d89e64aac66a768f93e1300dedc0df8ac9e7d27a57f
size 22963
