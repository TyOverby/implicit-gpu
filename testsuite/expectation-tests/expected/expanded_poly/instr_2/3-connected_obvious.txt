1 line segments

Line Segment 0 
1047 points
Clockwise? false
(59.0,104.81153)
(58.0,104.31153)
(57.376938,104.0)
(57.0,103.81153)
(56.0,103.31153)
(55.376938,103.0)
(55.0,102.81153)
(54.0,102.31153)
(53.376945,102.0)
(53.0,101.81153)
(52.0,101.31153)
(51.376938,101.0)
(51.0,100.81153)
(50.0,100.31153)
(49.376938,100.0)
(49.0,99.81153)
(48.0,99.31153)
(47.376938,99.0)
(47.0,98.81153)
(46.0,98.31153)
(45.376938,98.0)
(45.0,97.81153)
(44.0,97.31153)
(43.376938,97.0)
(43.0,96.81153)
(42.0,96.31153)
(41.376938,96.0)
(41.0,95.81153)
(40.0,95.31153)
(39.376945,95.0)
(39.0,94.81153)
(38.0,94.31153)
(37.376938,94.0)
(37.0,93.81153)
(36.0,93.31153)
(35.376938,93.0)
(35.0,92.81153)
(34.0,92.31153)
(33.376945,92.0)
(33.0,91.81153)
(32.0,91.31153)
(31.376938,91.0)
(31.0,90.81153)
(30.0,90.31153)
(29.38904,90.0)
(29.0,89.79905)
(28.0,89.255)
(27.554182,89.0)
(27.0,88.677414)
(26.0,88.065506)
(25.897404,88.0)
(25.0,87.41557)
(24.390654,87.0)
(24.0,86.72783)
(23.0,86.0)
(22.0,85.226875)
(21.71788,85.0)
(21.0,84.40785)
(20.523607,84.0)
(20.0,83.539375)
(19.40801,83.0)
(19.0,82.61729)
(18.363274,82.0)
(18.0,81.63673)
(17.382711,81.0)
(17.0,80.59199)
(16.460629,80.0)
(16.0,79.476395)
(15.59215,79.0)
(15.0,78.28212)
(14.773125,78.0)
(14.0,77.0)
(13.272173,76.0)
(13.0,75.609344)
(12.584425,75.0)
(12.0,74.10259)
(11.934491,74.0)
(11.322582,73.0)
(11.0,72.445816)
(10.744999,72.0)
(10.200948,71.0)
(10.0,70.61096)
(9.689231,70.0)
(9.20821,69.0)
(9.0,68.54167)
(8.757171,68.0)
(8.335051,67.0)
(8.0,66.15204)
(7.940599,66.0)
(7.5739126,65.0)
(7.233395,64.0)
(7.0,63.259903)
(6.9187512,63.0)
(6.629693,62.0)
(6.36531,61.0)
(6.1252394,60.0)
(6.0,59.42163)
(5.90922,59.0)
(5.7168913,58.0)
(5.5478473,57.0)
(5.401843,56.0)
(5.2786674,55.0)
(5.178144,54.0)
(5.100115,53.0)
(5.044467,52.0)
(5.0111113,51.0)
(5.0,50.0)
(5.0111113,49.0)
(5.044467,48.0)
(5.100115,47.0)
(5.178144,46.0)
(5.2786674,45.0)
(5.401843,44.0)
(5.5478473,43.0)
(5.7168913,42.0)
(5.90922,41.0)
(6.0,40.57837)
(6.1252394,40.0)
(6.36531,39.0)
(6.629693,38.0)
(6.9187512,37.0)
(7.0,36.740097)
(7.2333956,36.0)
(7.5739126,35.0)
(7.940599,34.0)
(8.0,33.847958)
(8.335051,33.0)
(8.757171,32.0)
(9.0,31.458326)
(9.20821,31.0)
(9.689231,30.0)
(10.0,29.38904)
(10.200948,29.0)
(10.744999,28.0)
(11.0,27.554182)
(11.322582,27.0)
(11.934491,26.0)
(12.0,25.897404)
(12.584425,25.0)
(13.0,24.390654)
(13.272173,24.0)
(14.0,23.0)
(14.773125,22.0)
(15.0,21.71788)
(15.59215,21.0)
(16.0,20.523607)
(16.460629,20.0)
(17.0,19.40801)
(17.382711,19.0)
(18.0,18.363274)
(18.363274,18.0)
(19.0,17.382711)
(19.40801,17.0)
(20.0,16.460629)
(20.523607,16.0)
(21.0,15.59215)
(21.71788,15.0)
(22.0,14.773125)
(23.0,14.0)
(24.0,13.272173)
(24.390654,13.0)
(25.0,12.584425)
(25.897404,12.0)
(26.0,11.934491)
(27.0,11.322582)
(27.554182,11.0)
(28.0,10.744999)
(29.0,10.200948)
(29.38904,10.0)
(30.0,9.689231)
(31.0,9.20821)
(31.458326,9.0)
(32.0,8.757171)
(33.0,8.335051)
(33.847958,8.0)
(34.0,7.940599)
(35.0,7.5739126)
(36.0,7.2333956)
(36.740097,7.0)
(37.0,6.9187512)
(38.0,6.629693)
(39.0,6.36531)
(40.0,6.1252394)
(40.57837,6.0)
(41.0,5.90922)
(42.0,5.7168913)
(43.0,5.5478473)
(44.0,5.401843)
(45.0,5.2786674)
(46.0,5.178144)
(47.0,5.100115)
(48.0,5.044467)
(49.0,5.0111113)
(50.0,5.0)
(51.0,5.0)
(52.0,5.0)
(53.0,5.0)
(54.0,5.0)
(55.0,5.0)
(56.0,5.0)
(57.0,5.0)
(58.0,5.0)
(59.0,5.0)
(60.0,5.0)
(61.0,5.0)
(62.0,5.0)
(63.0,5.0)
(64.0,5.0)
(65.0,5.0)
(66.0,5.0)
(67.0,5.0)
(68.0,5.0)
(69.0,5.0)
(70.0,5.0)
(71.0,5.0)
(72.0,5.0)
(73.0,5.0)
(74.0,5.0)
(75.0,5.0)
(76.0,5.0)
(77.0,5.0)
(78.0,5.0)
(79.0,5.0)
(80.0,5.0)
(81.0,5.0)
(82.0,5.0)
(83.0,5.0)
(84.0,5.0)
(85.0,5.0)
(86.0,5.0)
(87.0,5.0)
(88.0,5.0)
(89.0,5.0)
(90.0,5.0)
(91.0,5.0)
(92.0,5.0)
(93.0,5.0)
(94.0,5.0)
(95.0,5.0)
(96.0,5.0)
(97.0,5.0)
(98.0,5.0)
(99.0,5.0)
(100.0,5.0)
(101.0,5.0)
(102.0,5.0)
(103.0,5.0)
(104.0,5.0)
(105.0,5.0)
(106.0,5.0)
(107.0,5.0)
(108.0,5.0)
(109.0,5.0)
(110.0,5.0)
(111.0,5.0)
(112.0,5.0)
(113.0,5.0)
(114.0,5.0)
(115.0,5.0)
(116.0,5.0)
(117.0,5.0)
(118.0,5.0)
(119.0,5.0)
(120.0,5.0)
(121.0,5.0)
(122.0,5.0)
(123.0,5.0)
(124.0,5.0)
(125.0,5.0)
(126.0,5.0)
(127.0,5.0)
(128.0,5.0)
(129.0,5.0)
(130.0,5.0)
(131.0,5.0)
(132.0,5.0)
(133.0,5.0)
(134.0,5.0)
(135.0,5.0)
(136.0,5.0)
(137.0,5.0)
(138.0,5.0)
(139.0,5.0)
(140.0,5.0)
(141.0,5.0)
(142.0,5.0)
(143.0,5.0)
(144.0,5.0)
(145.0,5.0)
(146.0,5.0)
(147.0,5.0)
(148.0,5.0)
(149.0,5.0)
(150.0,5.0)
(151.0,5.0)
(152.0,5.0)
(153.0,5.0)
(154.0,5.0)
(155.0,5.0)
(156.0,5.0)
(157.0,5.0)
(158.0,5.0)
(159.0,5.0)
(160.0,5.0)
(161.0,5.0)
(162.0,5.0)
(163.0,5.0)
(164.0,5.0)
(165.0,5.0)
(166.0,5.0)
(167.0,5.0)
(168.0,5.0)
(169.0,5.0)
(170.0,5.0)
(171.0,5.0)
(172.0,5.0)
(173.0,5.0)
(174.0,5.0)
(175.0,5.0)
(176.0,5.0)
(177.0,5.0)
(178.0,5.0)
(179.0,5.0)
(180.0,5.0)
(181.0,5.0)
(182.0,5.0)
(183.0,5.0)
(184.0,5.0)
(185.0,5.0)
(186.0,5.0)
(187.0,5.0)
(188.0,5.0)
(189.0,5.0)
(190.0,5.0)
(191.0,5.0)
(192.0,5.0)
(193.0,5.0)
(194.0,5.0)
(195.0,5.0)
(196.0,5.0)
(197.0,5.0)
(198.0,5.0)
(199.0,5.0)
(200.0,5.0)
(201.0,5.0111113)
(202.0,5.044467)
(203.0,5.100115)
(204.0,5.178144)
(205.0,5.2786674)
(206.0,5.401843)
(207.0,5.5478473)
(208.0,5.7168913)
(209.0,5.90922)
(209.42163,6.0)
(210.0,6.1252394)
(211.0,6.36531)
(212.0,6.629693)
(213.0,6.9187512)
(213.2599,7.0)
(214.0,7.233395)
(215.0,7.5739126)
(216.0,7.940599)
(216.15204,8.0)
(217.0,8.335051)
(218.0,8.757171)
(218.54167,9.0)
(219.0,9.20821)
(220.0,9.689231)
(220.61096,10.0)
(221.0,10.200948)
(222.0,10.744999)
(222.44582,11.0)
(223.0,11.322582)
(224.0,11.934491)
(224.1026,12.0)
(225.0,12.584425)
(225.60934,13.0)
(226.0,13.272173)
(227.0,14.0)
(228.0,14.773125)
(228.28212,15.0)
(229.0,15.59215)
(229.4764,16.0)
(230.0,16.460629)
(230.592,17.0)
(231.0,17.382711)
(231.63673,18.0)
(232.0,18.363274)
(232.6173,19.0)
(233.0,19.40801)
(233.53937,20.0)
(234.0,20.523607)
(234.40785,21.0)
(235.0,21.71788)
(235.22688,22.0)
(236.0,23.0)
(236.72783,24.0)
(237.0,24.390654)
(237.41557,25.0)
(238.0,25.897404)
(238.0655,26.0)
(238.67741,27.0)
(239.0,27.554182)
(239.255,28.0)
(239.79906,29.0)
(240.0,29.38904)
(240.31078,30.0)
(240.7918,31.0)
(241.0,31.458326)
(241.24283,32.0)
(241.66495,33.0)
(242.0,33.847958)
(242.0594,34.0)
(242.42609,35.0)
(242.7675,36.0)
(243.0,36.697506)
(243.10083,37.0)
(243.43416,38.0)
(243.7675,39.0)
(244.0,39.697506)
(244.10083,40.0)
(244.43416,41.0)
(244.7675,42.0)
(245.0,42.697506)
(245.10083,43.0)
(245.43416,44.0)
(245.7675,45.0)
(246.0,45.697506)
(246.10083,46.0)
(246.43416,47.0)
(246.7675,48.0)
(247.0,48.697506)
(247.10083,49.0)
(247.43416,50.0)
(247.7675,51.0)
(248.0,51.69751)
(248.10083,52.0)
(248.43416,53.0)
(248.7675,54.0)
(249.0,54.697506)
(249.10083,55.0)
(249.43416,56.0)
(249.7675,57.0)
(250.0,57.697506)
(250.10083,58.0)
(250.43416,59.0)
(250.7675,60.0)
(251.0,60.69751)
(251.10083,61.0)
(251.43416,62.0)
(251.7675,63.0)
(252.0,63.69751)
(252.10083,64.0)
(252.43416,65.0)
(252.7675,66.0)
(253.0,66.6975)
(253.10083,67.0)
(253.43416,68.0)
(253.7675,69.0)
(254.0,69.6975)
(254.10083,70.0)
(254.43416,71.0)
(254.7675,72.0)
(255.0,72.69751)
(255.10083,73.0)
(255.43417,74.0)
(255.7675,75.0)
(256.0,75.69752)
(256.10083,76.0)
(256.43417,77.0)
(256.7675,78.0)
(257.0,78.69751)
(257.10083,79.0)
(257.43417,80.0)
(257.76752,81.0)
(258.0,81.6975)
(258.10083,82.0)
(258.43417,83.0)
(258.7675,84.0)
(259.0,84.69751)
(259.10083,85.0)
(259.43417,86.0)
(259.76752,87.0)
(260.0,87.6975)
(260.10083,88.0)
(260.43417,89.0)
(260.76752,90.0)
(261.0,90.6975)
(261.10083,91.0)
(261.43417,92.0)
(261.7675,93.0)
(262.0,93.69751)
(262.10083,94.0)
(262.43417,95.0)
(262.76752,96.0)
(263.0,96.6975)
(263.10083,97.0)
(263.43417,98.0)
(263.76752,99.0)
(264.0,99.6975)
(264.10083,100.0)
(264.43417,101.0)
(264.7675,102.0)
(265.0,102.69751)
(265.10083,103.0)
(265.43417,104.0)
(265.76752,105.0)
(266.0,105.6975)
(266.10083,106.0)
(266.43417,107.0)
(266.76752,108.0)
(267.0,108.6975)
(267.10083,109.0)
(267.43417,110.0)
(267.76752,111.0)
(268.0,111.69752)
(268.10083,112.0)
(268.43417,113.0)
(268.7675,114.0)
(269.0,114.69751)
(269.10083,115.0)
(269.43417,116.0)
(269.7675,117.0)
(270.0,117.69751)
(270.10083,118.0)
(270.43417,119.0)
(270.7675,120.0)
(271.0,120.69751)
(271.10083,121.0)
(271.43417,122.0)
(271.7675,123.0)
(272.0,123.69752)
(272.10083,124.0)
(272.43417,125.0)
(272.7675,126.0)
(273.0,126.69751)
(273.10083,127.0)
(273.43417,128.0)
(273.7675,129.0)
(274.0,129.69751)
(274.10083,130.0)
(274.43417,131.0)
(274.7675,132.0)
(275.0,132.69751)
(275.10083,133.0)
(275.43417,134.0)
(275.7675,135.0)
(276.0,135.69751)
(276.10083,136.0)
(276.43417,137.0)
(276.7675,138.0)
(277.0,138.69751)
(277.10083,139.0)
(277.43417,140.0)
(277.7675,141.0)
(278.0,141.69751)
(278.10083,142.0)
(278.43417,143.0)
(278.7675,144.0)
(279.0,144.69751)
(279.10083,145.0)
(279.43417,146.0)
(279.7675,147.0)
(280.0,147.69751)
(280.10083,148.0)
(280.43417,149.0)
(280.7675,150.0)
(281.0,150.69751)
(281.10083,151.0)
(281.43417,152.0)
(281.7675,153.0)
(282.0,153.69751)
(282.10083,154.0)
(282.43417,155.0)
(282.7675,156.0)
(283.0,156.69751)
(283.10083,157.0)
(283.43417,158.0)
(283.7675,159.0)
(284.0,159.69751)
(284.10083,160.0)
(284.43417,161.0)
(284.7675,162.0)
(285.0,162.69751)
(285.10083,163.0)
(285.43417,164.0)
(285.7675,165.0)
(286.0,165.69751)
(286.10083,166.0)
(286.43417,167.0)
(286.76752,168.0)
(287.0,168.69751)
(287.10083,169.0)
(287.43417,170.0)
(287.7675,171.0)
(288.0,171.69751)
(288.10083,172.0)
(288.43417,173.0)
(288.7675,174.0)
(289.0,174.69751)
(289.10083,175.0)
(289.43417,176.0)
(289.7675,177.0)
(290.0,177.69751)
(290.10083,178.0)
(290.43417,179.0)
(290.7675,180.0)
(291.0,180.69751)
(291.10083,181.0)
(291.43417,182.0)
(291.7675,183.0)
(292.0,183.69751)
(292.10083,184.0)
(292.43417,185.0)
(292.7666,186.0)
(293.0,186.7401)
(293.08124,187.0)
(293.3703,188.0)
(293.6347,189.0)
(293.87476,190.0)
(294.0,190.57837)
(294.0908,191.0)
(294.2831,192.0)
(294.45215,193.0)
(294.59814,194.0)
(294.72134,195.0)
(294.82187,196.0)
(294.89987,197.0)
(294.95554,198.0)
(294.9889,199.0)
(295.0,200.0)
(294.9889,201.0)
(294.95554,202.0)
(294.89987,203.0)
(294.82187,204.0)
(294.72134,205.0)
(294.59814,206.0)
(294.45215,207.0)
(294.2831,208.0)
(294.0908,209.0)
(294.0,209.42163)
(293.87476,210.0)
(293.6347,211.0)
(293.3703,212.0)
(293.08124,213.0)
(293.0,213.2599)
(292.7666,214.0)
(292.4261,215.0)
(292.0594,216.0)
(292.0,216.15204)
(291.66495,217.0)
(291.24283,218.0)
(291.0,218.54167)
(290.79178,219.0)
(290.31076,220.0)
(290.0,220.61096)
(289.79904,221.0)
(289.255,222.0)
(289.0,222.44582)
(288.67743,223.0)
(288.06552,224.0)
(288.0,224.1026)
(287.4156,225.0)
(287.0,225.60934)
(286.7278,226.0)
(286.0,227.0)
(285.22687,228.0)
(285.0,228.28212)
(284.40784,229.0)
(284.0,229.4764)
(283.53937,230.0)
(283.0,230.592)
(282.61728,231.0)
(282.0,231.63673)
(281.63672,232.0)
(281.0,232.6173)
(280.59198,233.0)
(280.0,233.53937)
(279.47638,234.0)
(279.0,234.40785)
(278.28214,235.0)
(278.0,235.22688)
(277.0,236.0)
(276.0,236.72783)
(275.60934,237.0)
(275.0,237.41557)
(274.1026,238.0)
(274.0,238.0655)
(273.0,238.67741)
(272.44583,239.0)
(272.0,239.255)
(271.0,239.79906)
(270.61096,240.0)
(270.0,240.31078)
(269.0,240.7918)
(268.5417,241.0)
(268.0,241.24283)
(267.0,241.66495)
(266.15204,242.0)
(266.0,242.0594)
(265.0,242.42609)
(264.0,242.7666)
(263.2599,243.0)
(263.0,243.08125)
(262.0,243.3703)
(261.0,243.63469)
(260.0,243.87476)
(259.42163,244.0)
(259.0,244.09077)
(258.0,244.28311)
(257.0,244.45215)
(256.0,244.59816)
(255.0,244.72133)
(254.0,244.82185)
(253.0,244.89989)
(252.0,244.95554)
(251.0,244.98889)
(250.0,245.0)
(249.0,244.98889)
(248.0,244.95554)
(247.0,244.89989)
(246.0,244.82185)
(245.0,244.72133)
(244.0,244.59816)
(243.0,244.45215)
(242.0,244.28311)
(241.0,244.09077)
(240.57837,244.0)
(240.0,243.87476)
(239.0,243.63469)
(238.0,243.3703)
(237.0,243.08125)
(236.7401,243.0)
(236.0,242.7666)
(235.0,242.42609)
(234.0,242.0594)
(233.84796,242.0)
(233.0,241.66495)
(232.0,241.24283)
(231.45833,241.0)
(231.0,240.7918)
(230.0,240.31078)
(229.38904,240.0)
(229.0,239.79906)
(228.0,239.255)
(227.55418,239.0)
(227.0,238.67741)
(226.0,238.0655)
(225.8974,238.0)
(225.0,237.41557)
(224.39066,237.0)
(224.0,236.72783)
(223.0,236.0)
(222.0,235.22688)
(221.71788,235.0)
(221.0,234.40785)
(220.5236,234.0)
(220.0,233.53937)
(219.408,233.0)
(219.0,232.6173)
(218.36327,232.0)
(218.0,231.63962)
(217.36038,231.0)
(217.0,230.63962)
(216.36038,230.0)
(216.0,229.63962)
(215.36038,229.0)
(215.0,228.63962)
(214.36038,228.0)
(214.0,227.63962)
(213.36038,227.0)
(213.0,226.63962)
(212.36038,226.0)
(212.0,225.63962)
(211.36038,225.0)
(211.0,224.63962)
(210.36038,224.0)
(210.0,223.63962)
(209.36038,223.0)
(209.0,222.63962)
(208.36038,222.0)
(208.0,221.63962)
(207.36038,221.0)
(207.0,220.63962)
(206.36038,220.0)
(206.0,219.63962)
(205.36038,219.0)
(205.0,218.63962)
(204.36038,218.0)
(204.0,217.63962)
(203.36038,217.0)
(203.0,216.63962)
(202.36038,216.0)
(202.0,215.63962)
(201.36038,215.0)
(201.0,214.63962)
(200.36038,214.0)
(200.0,213.63962)
(199.36038,213.0)
(199.0,212.63962)
(198.36038,212.0)
(198.0,211.63962)
(197.36038,211.0)
(197.0,210.63962)
(196.36038,210.0)
(196.0,209.63962)
(195.36038,209.0)
(195.0,208.63962)
(194.36038,208.0)
(194.0,207.63962)
(193.36038,207.0)
(193.0,206.63962)
(192.36038,206.0)
(192.0,205.63962)
(191.36038,205.0)
(191.0,204.63962)
(190.36038,204.0)
(190.0,203.63962)
(189.36038,203.0)
(189.0,202.63962)
(188.36038,202.0)
(188.0,201.63962)
(187.36038,201.0)
(187.0,200.63962)
(186.36038,200.0)
(186.0,199.63962)
(185.36038,199.0)
(185.0,198.63962)
(184.36038,198.0)
(184.0,197.63962)
(183.36038,197.0)
(183.0,196.63962)
(182.36038,196.0)
(182.0,195.63962)
(181.36038,195.0)
(181.0,194.63962)
(180.36038,194.0)
(180.0,193.63962)
(179.36038,193.0)
(179.0,192.63962)
(178.36038,192.0)
(178.0,191.63962)
(177.36038,191.0)
(177.0,190.63962)
(176.36038,190.0)
(176.0,189.63962)
(175.36038,189.0)
(175.0,188.63962)
(174.36038,188.0)
(174.0,187.63962)
(173.36038,187.0)
(173.0,186.63962)
(172.36038,186.0)
(172.0,185.63962)
(171.36038,185.0)
(171.0,184.63962)
(170.36038,184.0)
(170.0,183.63962)
(169.36038,183.0)
(169.0,182.63962)
(168.36038,182.0)
(168.0,181.63962)
(167.36038,181.0)
(167.0,180.63962)
(166.36038,180.0)
(166.0,179.63962)
(165.36038,179.0)
(165.0,178.63962)
(164.36038,178.0)
(164.0,177.63962)
(163.36038,177.0)
(163.0,176.63962)
(162.36038,176.0)
(162.0,175.63962)
(161.36038,175.0)
(161.0,174.63962)
(160.36038,174.0)
(160.0,173.63962)
(159.36038,173.0)
(159.0,172.63962)
(158.36038,172.0)
(158.0,171.63962)
(157.36038,171.0)
(157.0,170.63962)
(156.36038,170.0)
(156.0,169.63962)
(155.36038,169.0)
(155.0,168.63962)
(154.36038,168.0)
(154.0,167.63962)
(153.36038,167.0)
(153.0,166.63962)
(152.36038,166.0)
(152.0,165.63962)
(151.36038,165.0)
(151.0,164.63962)
(150.36038,164.0)
(150.0,163.63962)
(149.36038,163.0)
(149.0,162.63962)
(148.36038,162.0)
(148.0,161.63962)
(147.36038,161.0)
(147.0,160.63962)
(146.36038,160.0)
(146.0,159.63962)
(145.36038,159.0)
(145.0,158.63962)
(144.36038,158.0)
(144.0,157.6396)
(143.3604,157.0)
(143.0,156.6396)
(142.3604,156.0)
(142.0,155.6396)
(141.3604,155.0)
(141.0,154.6396)
(140.36038,154.0)
(140.0,153.63962)
(139.36038,153.0)
(139.0,152.63962)
(138.36038,152.0)
(138.0,151.6396)
(137.3604,151.0)
(137.0,150.6396)
(136.3604,150.0)
(136.0,149.6396)
(135.3604,149.0)
(135.0,148.6396)
(134.36038,148.0)
(134.0,147.63962)
(133.36038,147.0)
(133.0,146.63962)
(132.36038,146.0)
(132.0,145.6396)
(131.3604,145.0)
(131.0,144.6396)
(130.3604,144.0)
(130.0,143.6396)
(129.3604,143.0)
(129.0,142.6396)
(128.3604,142.0)
(128.0,141.63962)
(127.36039,141.0)
(127.0,140.63962)
(126.36039,140.0)
(126.0,139.6396)
(125.3604,139.0)
(125.0,138.6396)
(124.3604,138.0)
(124.0,137.6396)
(123.27153,137.0)
(123.0,136.81152)
(122.0,136.31152)
(121.376945,136.0)
(121.0,135.81152)
(120.0,135.31152)
(119.376945,135.0)
(119.0,134.81152)
(118.0,134.31152)
(117.376945,134.0)
(117.0,133.81152)
(116.0,133.31152)
(115.376945,133.0)
(115.0,132.81152)
(114.0,132.31152)
(113.376945,132.0)
(113.0,131.81152)
(112.0,131.31152)
(111.376945,131.0)
(111.0,130.81152)
(110.0,130.31152)
(109.376945,130.0)
(109.0,129.81152)
(108.0,129.31152)
(107.376945,129.0)
(107.0,128.81152)
(106.0,128.31152)
(105.376945,128.0)
(105.0,127.81153)
(104.0,127.31153)
(103.376945,127.0)
(103.0,126.81153)
(102.0,126.31153)
(101.376945,126.0)
(101.0,125.81153)
(100.0,125.31153)
(99.376945,125.0)
(99.0,124.81153)
(98.0,124.31153)
(97.376945,124.0)
(97.0,123.81153)
(96.0,123.31153)
(95.376945,123.0)
(95.0,122.81153)
(94.0,122.31153)
(93.376945,122.0)
(93.0,121.81153)
(92.0,121.31153)
(91.376945,121.0)
(91.0,120.81153)
(90.0,120.31153)
(89.376945,120.0)
(89.0,119.81153)
(88.0,119.31153)
(87.376945,119.0)
(87.0,118.81153)
(86.0,118.31153)
(85.376945,118.0)
(85.0,117.81153)
(84.0,117.31153)
(83.376945,117.0)
(83.0,116.81153)
(82.0,116.31153)
(81.376945,116.0)
(81.0,115.81153)
(80.0,115.31153)
(79.376945,115.0)
(79.0,114.81153)
(78.0,114.31153)
(77.376945,114.0)
(77.0,113.81153)
(76.0,113.31153)
(75.376945,113.0)
(75.0,112.81153)
(74.0,112.31153)
(73.376945,112.0)
(73.0,111.81153)
(72.0,111.31153)
(71.376945,111.0)
(71.0,110.81153)
(70.0,110.31153)
(69.376945,110.0)
(69.0,109.81153)
(68.0,109.31153)
(67.376945,109.0)
(67.0,108.81153)
(66.0,108.31153)
(65.376945,108.0)
(65.0,107.81153)
(64.0,107.31153)
(63.376945,107.0)
(63.0,106.81153)
(62.0,106.31153)
(61.376945,106.0)
(61.0,105.81153)
(60.0,105.31153)
(59.376945,105.0)
